packages/sw-precaching/src/lib/controllers/base-cache-manager.js

```diff
@@ -13,7 +13,8 @@ class BaseCacheManager {
    */
   constructor(cacheName) {
     this._entriesToCache = new Map();
-    this._cacheName = cacheName;
+    this._requestWrapper = new RequestWrapper({cacheName});
```

Comment:
Possible stupid question, is there a reason for this not to be the default option?